src/utils/utils.ts

```diff
@@ -1,6 +1,8 @@
 import { LOCALES } from "./constants";
 import { snakeCase } from "lodash";
 
+export * from "./urlUtils";
+
```

Comment:
Why here and not in `src/utils/index.ts`?

Reply:
I actually missed that, my bad. Will put this line in `src/utils/index.ts`.

Side-note: the filename of `src/utils/utils.ts` is not ideal. It's currently a collection of miscellaneous functions. I don't have a good alternative in mind right now, though.